nautobot/extras/models/datasources.py

```diff
@@ -67,9 +67,8 @@ class Meta:
         verbose_name_plural = "Git repositories"
 
     def __init__(self, *args, **kwargs):
-        # If instantiated from the REST API, the originating Request will be passed as a kwarg:
-        # FIXME(jathan); Replace this with user instead; make request go away.
-        self.request = kwargs.pop("request", None)
+        # If instantiated from the API/UI, the originating User will be passed as a kwarg:
```

Comment:
This comment isn't entirely true. Other callers have been setting this as an instance variable.